tasks/utils/gnosis.js

```diff
@@ -80,6 +82,7 @@ const getDelegates = async (safe, url) =>
 const api = (safe, network) => {
   const url = getUrl(network);
   return {
+    endpoint: url,
```

Comment:
Why did you add this line, if it wasn't in the last iteration? Just for my own understanding

Reply:
there was no way to get the API url, before this, now using this property the user can get the endpoint.